Documentation/supermuc.rst

```diff
@@ -34,24 +40,52 @@ where ddddd is an arbitrary 5-digital port number.
     
 With ddddd the same port number as before.
 
-4. Create SSH key by typing (use a non-empty passphrase, not too long as you will need to type it often)
+3. Create SSH key by typing (use a non-empty passphrase, not too long as you will need to type it often)
 
 ::
 
   ssh-keygen -t rsa 
 
-5. Go to https://github.com/settings/ssh, add a new SSH key, pasting the public key you just created on supermuc  ~/.ssh/id_rsa.pub. 
-Logout of supermuc and log back in (ssh supermucNG). You should now be able to clone SeisSol including the submodules using:
+4. Go to https://github.com/settings/ssh, add a new SSH key, paste the public SSH key you just created on supermuc  ~/.ssh/id_rsa.pub. For example,
```

Comment:
4. Go to https://github.com/settings/ssh, add a new SSH key, and paste the public SSH key you just created (the content of ~/.ssh/id_rsa.pub on supermucNG).
Remove Everything from "For example... to L68".

5. Logout of supermucNG and log back in. You should now be able to clone SeisSol including the submodules using: